eslint-configs/eslint-config-seekingalpha-typescript/rules/typescript-eslint/index.js

```diff
@@ -0,0 +1,389 @@
+module.exports = {
+  plugins: ['@typescript-eslint/eslint-plugin'],
+
+  rules: {
+
+    'import/extensions': [
+      'error',
+      'ignorePackages',
+      {
+        js: 'never',
+        jsx: 'never',
+        ts: 'never',
+        tsx: 'never',
+      },
+    ],
+
+    '@typescript-eslint/adjacent-overload-signatures': 'error',
+
+    '@typescript-eslint/await-thenable': 'error',
+
+    '@typescript-eslint/ban-ts-comment': 'error',
+
+    '@typescript-eslint/ban-tslint-comment': 'error',
+
+    '@typescript-eslint/ban-types': 'error',
+
+    'brace-style': 'off',
```

Comment:
plz move all the native eslint overrides in a separate file and include that file in `eslint-configs/eslint-config-seekingalpha-typescript/index.js` inside `extend` option